src/NServiceBus.Core/Routing/AssemblyRouteSource.cs

```diff
@@ -20,7 +20,7 @@ public AssemblyRouteSource(Assembly messageAssembly, UnicastRoute route)
         public IEnumerable<RouteTableEntry> GenerateRoutes(Conventions conventions)
         {
             var routes = messageAssembly.GetTypes()
-                .Where(t => conventions.IsMessageType(t))
+                .Where(conventions.IsMessageType)
```

Comment:
I didn't look to see if these were one-and-done or called multiple times, so if they are just called once, then yeah it doesn't make much difference.